test/ssltestlib.c

```diff
@@ -12,6 +12,13 @@
 #include "internal/nelem.h"
 #include "ssltestlib.h"
 #include "testutil.h"
+#ifndef OPENSSL_NO_KTLS
+#include "internal/ktls.h"
+#include "internal/sockets.h"
```

Comment:
Why do we need these internal headers here?

Reply:
We don't need them. I'll remove these leftovers which I added while playing around with possible implementations of the test.